src/translations/en.json

```diff
@@ -3403,7 +3403,7 @@
                 "conf_mode": "Configuration method",
                 "default": "Default",
                 "require_admin": "Admin only",
-                "sidebar": "Show in sidebar",
+                "sidebar": "In sidebar",
```

Comment:
"Added to sidebar" makes a bit more sense to me but it would make it longer which is not great for a column name